tests/pfcwd/test_pfcwd_all_port_storm.py

```diff
@@ -19,6 +19,12 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope="class")
+def pfc_queue_idx():
+    # Needed for start_background_traffic
+    yield 3   # Hardcoded in the testcase as well.
```

Comment:
why hard code here?

Reply:
@kevinskwang , as the comment in the code says, the priority is hard coded in the testcase as well.